Makefile

```diff
@@ -468,6 +469,32 @@ RECSET_SHARED_CSS_DEPS=$(CSS)/vendor/bootstrap.min.css \
 RECSET_CSS_SOURCE=$(COMMON)/styles/app.css \
 	$(COMMON)/vendor/MarkdownEditor/styles/github-markdown.css
 
+DEFAULT_JS_DEPS=$(JS)/vendor/angular.js \
```

Comment:
Eventually, when we get to refactor the makefile, most of the JS_DEPS can be changed to this one.  This is a lot cleaner than having multiple sources of JS.